contrib/init/systemd/docker.service

```diff
@@ -5,7 +5,8 @@ After=network.target docker.socket
 Requires=docker.socket
 
 [Service]
-ExecStart=/usr/bin/docker -d -H fd://
+EnvironmentFile=/etc/default/docker
```

Comment:
This is a painful experience: running 1.6 with `/etc/sysconfig/docker` on Fedora 20.  Upgrade to 1.7.1, suddenly the daemon options are not respected.  Go hunting for why it broke, find out that it was done "for my own good" "because systemd" here and in https://github.com/docker/docker/pull/14513

This was not in the 1.7 release notes, either, as far as I know at https://github.com/docker/docker/blob/master/CHANGELOG.md

It's a pretty big change so please consider revising the release notes at least.

The proper way to handle this would have been to deprecate `/etc/sysconfig/docker` and the other locations listed above.  Log a warning if they are used; point to the docs; mention it in the release notes.  But don't break working setups until we've had a chance to fix them.  That's a far bigger disappointment than the one @tianon mentions.